### DISCUSSION TITLE
Open dropdown upwards

### DISCUSSION THREAD

**Discussion by bruninoit** [csandman/chakra-react-select#328] [Q&A] [answered]
Hi, is there a way to make the select open at the top when it's at the bottom of the screen?

**Comment by csandman** [comment, marked as answer]
Yeah, you can set the `menuPlacement` prop to `"auto"`:

> ### menuPlacement
> 
> Default placement of the menu in relation to the control. 'auto' will flip when there isn't enough space below the control.
> 
> ```
> One of<
> 	"bottom",
> 	"auto",
> 	"top"
> >
> ```

— https://react-select.com/props

The default value for it is `"bottom"`.
